docs/faq.md

```diff
@@ -139,6 +139,10 @@ If you want to change the behavior of the mounts, that will also require [additi
 New-NetFirewallRule -DisplayName "WSL" -Direction Inbound -InterfaceAlias "vEthernet (WSL)" -Action Allow
 ```
 
+#### **Q: Can I map `host.docker.internal` to `host-gateway` with the flag `--add-host`?
+
+**A:** No. The special value `host-gateway` is something Docker Desktop specific and is not supported in Rancher Desktop yet. However, you can access services running on the host machine from within a container via `host.docker.internal` without passing the `--add-host` flag.
```

Comment:
Should we also mention `host.rancher-desktop.internal`?

See [here](https://github.com/rancher-sandbox/rancher-desktop/blob/e313649b5f544a59cbce9ecc69f975431653e233/pkg/rancher-desktop/backend/wsl.ts#L142).

Reply:
@Nino-K , I included `host.rancher-desktop.internal` in the FAQ item. Please take a look.